pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/ParseTreeConverter.java

```diff
@@ -2412,6 +2412,28 @@ private String getSqlTypeNameFromCode(int sqlTypeCode) {
         return ret;
     }
 
+    private String getColumnNameInSelectList(ColumnInfo ci) {
+        String colName = ci.colName;
+        assert colName != null;
+
+        int dotIdx = colName.lastIndexOf(".");
+        if (dotIdx > 0
+                && dotIdx < colName.length() - 1
+                && ci.className != null
+                && ci.className.length() > 0
+                && ci.attrName != null) {
+
+            String afterDot = colName.substring(dotIdx + 1).toLowerCase();
+            String attrName = ci.attrName.toLowerCase();
+            if (afterDot.equals(attrName)) {
```

Comment:
```suggestion
            String afterDot = colName.substring(dotIdx + 1);
            if (afterDot.equalsIgnoreCase(ci.attrName)) {
```

equalsIgnoreCase is way faster because it doesn't copy objects.

Reply:
Thank you